### ISSUE TITLE
Not able to open non official Roms

### ISSUE THREAD

**Issue by HJMBgit84** [rafaelvcaetano/melonDS-android#174]
Hi,
first thanks for the great update 1.3.0 made in the emulator :), the issue atm is that I have a version of zelda phantom hourglass and zelda Spirit tracks both of them are rom Dpad Patched and not able to play them, I can use the original roms but when trying with the other than that has Dpad Patched to be able to use Dpad for the character the emulator says error.

Thanks a lot for your work and let me know if more info is required.

**Comment by rafaelvcaetano** [comment]
I've never heard of such mod. What does the patch do? Can you point me to somewhere to learn more about it?

**Comment by HJMBgit84** [comment]
Check it out here and go to minute 01:30 to see it directly and if you want in the video description is the link of the game to try it:
https://youtu.be/EaEM-Ti_scM
There are modded Dpad versions for Spirit tracks and phantom hourglass :)

**Comment by HJMBgit84** [comment]
I think we can close this issue I tested the 1.4.0 and now the d-pad patched versions runs fine do you consider that is necessary to keep it open?

**Comment by rafaelvcaetano** [comment]
I didn't make any particular change that could have fixed this, so that's weird. Leave it open for now until I take a proper look at what was happening.

**Comment by HJMBgit84** [comment]
Got it, ok it will remain open, thanks for your work :)

**Comment by HJMBgit84** [comment]
I think the issue is with the DSi system, booting with DS normal it works but with DSi shows the error, tested on the 1.5.0 released today.

**Comment by WaluigiWare64** [comment]
Yes, the DSi has restrictions in place to only boot official carts, however, this can be circumvented by installing CFW on the emulated DSi.
